analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -372,13 +372,12 @@ open class Npm(
         )
     }
 
+    private fun File.isSymlink(): Boolean = realFile() != this
```

Comment:
How about putting this into `Extensions.kt` instead, just like `File.realFile()`?

Reply:
Also, I just realized a new possible issue now that this can operate on arbitrary files, not just on files returned by `listFiles()`: `realFile()` always returns an absolute path, but `this` may be relative, so the comparison should be changed to `realFile() != absoluteFile`.

Reply:
Good idea! Done.